### PR TITLE
Tab styling to match designs

### PR DESCRIPTION
**Pull request by julio8a**
- match Sketch library design
- Added Calcite global CSS variables
- Updated calcite-fonts to 1.3.2 (fixed jumping font when switching weights)

### CODE REVIEW

File: src/assets/styles/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    --calcite-global-ui-foreground-hover: #{$blk-010};
+    --calcite-global-ui-foreground-pressed: #{$blk-020};
+    --calcite-global-ui-text-1: #{$blk-220};
+    --calcite-global-ui-text-2:#{$blk-170};
+    --calcite-global-ui-text-3: #{$blk-140};
+
+  }
+  
+  :host([theme="dark"]) {
+    --calcite-global-ui-blue: #00A0FF;
```
Comment (paulcpederson) [inline src/assets/styles/_colors.scss]: shouldn't these hard-coded hex's use the `-dark` sass vars?
https://github.com/Esri/calcite-colors/blob/master/colors.scss#L485

Comment (julio8a) [reply src/assets/styles/_colors.scss]: @bstifle, this is where we would need to update them:
https://github.com/Esri/calcite-colors/blob/master/colors.scss#L485

File: src/assets/styles/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
+:root {
```
Comment (paulcpederson) [inline src/assets/styles/_colors.scss]: @oknoway not sure if you did your mapping for converting to calcite-colors 2.0, if not this may be helpful

**Comment by bstifle** [comment]
@paulcpederson  


Kyle H. and I worked on these dark theme colors. Kyle was going to wrap those into the calcite colors, I think. not sure where that is at. It was going to have its own prefix, such as

dt-yy-000
dt-bb-000
dt-gg-000
dt-rr-000

or something along those lines.

Lets merge this thing and update those colors in a different issue

**Comment by bstifle** [comment]
OK yeah @julio8a  @paulcpederson just found Kyle and mine's conversation about the DT colors. 

convention:
d-rr-410
d-rr-420
d-rr-430
 
d-bb-410
d-bb-420
d-bb-430

waiting for his response to see where he left off on that. will update after

File: src/components/calcite-tab-title/calcite-tab-title.scss [hunk 3]
```diff
@@ -9,29 +9,28 @@
   a {
     outline: none;
     text-decoration: none;
-    color: var(--calcite-tabs-color-active);
```
Comment (paulcpederson) [inline src/components/calcite-tab-title/calcite-tab-title.scss]: I think these were meant to allow people to theme the tab component. Will wait for @patrickarlt to weigh in.

**Comment by paulcpederson** [comment]
I don't think you're necessarily intending to do this, but I really like the move of making all of our components key off the same css variables. If we shared the css vars across all components, it would be easy to completely theme the entire set of components. You could still theme just one element by doing something like: 

```
calcite-tabs {
  --calcite-global-ui-blue: purple;
}
```

**Comment by julio8a** [comment]
@paulcpederson, yes that's the intent. :)

File: src/assets/styles/_colors.scss (hunk 1 above)
Comment (julio8a) [reply src/assets/styles/_colors.scss]: This should now be updated.

**Comment by paulcpederson** [comment]
Beautiful! new vars look good. I'm just going to merge this on in there.

**Pull request merged**